conf/docker/solr/Dockerfile

```diff
@@ -1,19 +1,31 @@
-FROM centos:7.2.1511
+FROM centos:7
 MAINTAINER Dataverse (support@dataverse.org)
 
-RUN yum install -y wget unzip perl git java-1.8.0-openjdk-devel postgresql.x86_64
+RUN yum install -y unzip java-1.8.0-openjdk-devel postgresql.x86_64 lsof ncat crontab
```

Comment:
are you using all the packages?

We probably wouldn't want to ever run cron this way.  Using a kube cron job would probably be the right approach when we need it.